src/erlcloud_ddb2.erl

```diff
@@ -3241,7 +3274,8 @@ restore_table_from_backup(BackupArn, TargetTableName, Opts, Config)
      Config,
      "DynamoDB_20120810.RestoreTableFromBackup",
      [{<<"BackupArn">>, BackupArn},
-      {<<"TargetTableName">>, TargetTableName}]),
+      {<<"TargetTableName">>, TargetTableName}],
+      DdbOpts),
```

Comment:
Looks like there is a small alignment issue here, the `DdbOpts` should align to the `[` on L3276.

```suggestion
     DdbOpts),
```